api/repositories/service_binding_repository_test.go

```diff
@@ -1149,6 +1155,174 @@ var _ = Describe("ServiceBindingRepo", func() {
 		})
 	})
 
+	Describe("GetServiceBindingParameters", func() {
+		var (
+			serviceBindingGUID   string
+			searchGUID           string
```

Comment:
I think you could be using the `serviceBindingGUID` instead of having to have a `searchGUID` here. In the context below where you set the `searchGUID`, you could just set `serviceBindingGUID` to `i-do-not-exist` - I think it should have the same effect.